galaxy_ng/app/viewsets.py

```diff
@@ -0,0 +1,15 @@
+from pulpcore.plugin import viewsets as pulp_viewsets
+from rest_framework import mixins
+
+from galaxy_ng.app import models
+from galaxy_ng.app.api.ui import serializers
+
+
+class ContainerDistributionViewSet(
```

Comment:
I had to create this Viewset here, even not being used is necessary so Pulp can dynamically import it during the removal tasks, it needs it to generate the URL.